### ISSUE TITLE
AC4236 missing sensors after update

### ISSUE THREAD

**Issue by muuuh** [kongo09/philips-airpurifier-coap#136]
Hello,

thank you for your integration. After the update v0.10.9 => v0.18.7 <s>I don't</s> didn't see the following sensors anymore:
- sensor.fan_pm2_5
- sensor.fan_indoor_allergen_index
- sensor.fan_total_volatile_organic_compounds

I have
<ul>
<li>
<details>
  <summary>fan.fan</summary>
<pre>
preset_modes: auto, sleep, allergy sleep, speed 1, speed 2, turbo
percentage: 50
percentage_step: 25
preset_mode: speed 1
name: redacted
type: AC4236
model_id: AC4236/14
product_id: redacted
device_id: redacted
device_version: 4.1.8
software_version: Ms4418
wifi_version: AWS_Philips_AIR@75.3
error_code: 0
language: EN
preferred_index: 1
runtime: redacted
icon: pap:speed_1
friendly_name: redacted
supported_features: 9
</pre></details></li>
<li>light.fan_display_backlight</li>
<li>light.fan_light_brightness</li>
<li>select.fan_preferred_index</li>
<li>switch.fan_child_lock</li></ul>

**Workaround:**

I noticed an error (undefined?) concerning EXTRA_SENSOR_TYPES https://github.com/kongo09/philips-airpurifier-coap/blob/4312bc8322180bf63c62101a4be58bd059e435f8/custom_components/philips_airpurifier_coap/sensor.py#L28
and tried to remove this line and removed the following lines https://github.com/kongo09/philips-airpurifier-coap/blob/4312bc8322180bf63c62101a4be58bd059e435f8/custom_components/philips_airpurifier_coap/sensor.py#L75-L77

This returned the sensor and I only have the following error now:
```
Entity sensor.fan_total_volatile_organic_compounds (<class 'custom_components.philips_airpurifier_coap.sensor.PhilipsSensor'>) is using native unit of measurement 'None' which is not a valid unit for the device class ('volatile_organic_compounds') it is using; expected one of ['µg/m³']; Please update your configuration if your entity is manually configured, otherwise create a bug report at https://github.com/kongo09/philips-airpurifier-coap/issues
```
which seems to be a minor error. :smile:

**Comment by igita** [comment]
+1 - same here

I got this error: 

Unable to prepare setup for platform 'philips_airpurifier_coap.sensor': Platform not found (cannot import name 'EXTRA_SENSOR_TYPES' from 'custom_components.philips_airpurifier_coap.const' (/home/homeassistant/.homeassistant/custom_components/philips_airpurifier_coap/const.py)).

After re adding my AC2889 - I only can see 1 device with 4 entities
would be great if it could be fixed again.

I am on Homeassistant 2024.3.3

**Comment by kongo09** [comment]
should be fixed in v0.18.8

sorry for messing it up
